hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestZKAsyncRegistry.java

```diff
@@ -128,4 +130,42 @@ public void testNoMetaAvailable() throws InterruptedException {
       }
     }
   }
+
+  /**
+   * Test meta tablestate implementation.
+   * Test is a bit involved because meta has replicas... Replica assign lags so check
+   * between steps all assigned.
+   */
+  @Test
+  public void testMetaTableState() throws IOException, ExecutionException, InterruptedException {
+    assertTrue(TEST_UTIL.getMiniHBaseCluster().getMaster().isActiveMaster());
+    while (!TEST_UTIL.getMiniHBaseCluster().getMaster().isInitialized()) {
```

Comment:
q: Isn't this guaranteed by startMiniCluster()? It doesn't return until there is an active initialized master and it can scan meta. If so, can be omitted?

Reply:
Leftover from debug. Purged.